bounce/server/resource/user.py

```diff
@@ -76,3 +76,68 @@ class GetUserResponse(metaclass=ResourceMeta):
             },
         }
     }
+
+
+class SearchUsersRequest(metaclass=ResourceMeta):
+    """Defines the schema for a GET /users/search request."""
+    __params__ = {
+        'query': {
+            'type': 'string',
```

Comment:
Minor: You might want to add a `minimum: 0` to some of these fields.

Reply:
done.  Set minimum to 0 for page and size

Reply:
we should also do this for clubs btw